js/bundles/modules/data.js

```diff
@@ -9,7 +9,7 @@ Object.defineProperty(DevExpress.data, 'errorHandler', {
         return require('../../data/errors').errorHandler;
     },
     set: function(value) {
-        errors.log('W0003', 'DevExpress.data', 'errorHandler', '21.1', 'Use the \'setErrorHandler\' method instead');
+        errors.log('W0003', 'DevExpress.data', 'errorHandler', '21.2', 'Use the \'setErrorHandler\' method instead');
```

Comment:
Is this a version when a method/property was deprecated? Should we change it?

Reply:
fixed